<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataReplay_Form</class>
 <widget class="QWidget" name="DataReplay_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,9" columnstretch="2,0,8">
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数据回放</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_plot"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="treeWidget_datafile">
     <column>
      <property name="text">
       <string>数据</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择文件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filepath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_selectfile">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_plot">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
